tile-group.lisp

```diff
@@ -1052,6 +1052,39 @@ space."
             (show-frame-indicator group))
           (run-hook-with-args *remove-split-hook* l frame)))))
 
+(defcommand (remove-sibling tile-group)
+    (&optional (group (current-group))
+	       (frame (let ((window-type (type-of (current-window))))
+			           (if (eq window-type 'float-window)
+			               nil ;; (focus-all (current-window))
+			               (window-frame (current-window))))))
+    ()
```

Comment:
Is there a reason we have such a complicated (&optional) form for the arguments?

Reply:
I had originally used a function (current-frame), but realized as I was submitting the pull request that that function was in my init file, not stumpwm itself.